app/models/sketch.rb

```diff
@@ -54,14 +54,21 @@ def update_boards_metadata
         Board.find_or_create_by(board)
       end
       # clear boards metadata
-      Board.for_user(user_id).find_by(mac: board['mac']).clear_boards_metadata
+      Board.for_user(user_id).find_by(mac: board['mac'])&.clear_boards_metadata
```

Comment:
ce face &  ?  e ca sa nu dea eroare in caz ca e null ?  gen sa il trateze ca false si sa nu mai faca clearu ?